README.md

```diff
@@ -27,6 +27,16 @@ Information: [This plugin is also available on Gradle plugins](https://plugins.g
 
 Snapshots can be found [here](https://oss.sonatype.org/#nexus-search;quick~gradle-maven-publish-plugin).
 
+### Variants
```

Comment:
Could you move this further down? Between `Settings properties` and `Where to upload to`. I'd also add Android to the title because this plugin is not just used by Android libraries